pkg/apis/beat/v1beta1/beat_types.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // BeatSpec defines the desired state of a Beat.
 type BeatSpec struct {
-	// Type is the type of the Beat to deploy (filebeat, metricbeat, etc.). Any string can be used,
+	// Type is the type of the Beat to deploy (filebeat, metricbeat, heartbeat, etc.). Any string can be used,
 	// but well-known types will be recognized and will allow to provide sane default configurations.
```

Comment:
```suggestion
	// but well-known types will be provided with default configurations
```

not in this PR but noticed this could be clarified

Reply:
Good catch. I think we should reword it even more, so we don't mention default configurations at all.

Reply:
I think you're right, I was trying to think of how to say "you don't need to specify the image"